### PR TITLE
some tracking fixes

### PR DESCRIPTION
**Pull request by aholdahl**

### CODE REVIEW

**Comment by dhruv1794** [comment]
These changes look good.

**Comment by aholdahl** [comment]
@dhruv1794 do you have the correct permission to formally approve?

**Comment by dhruv1794** [comment]
@aholdahl I cant approve the changes. I can just review them.

File: src/classBased/sezzle.js [hunk 1]
```diff
@@ -31,10 +31,10 @@ class SezzleJS {
    * Also logs initGTMScript event if condition matches
    */
   async initializeWidget() {
-    Utils.logEvent('sezzle-widget-request', this._configInst);
+    Utils.logEvent('request', this._configInst);
     await this._loadCSS();
     this._renderAwesomeSezzle.initializeRendering();
-    if(document.sezzleDefaultModalVersion === "sezzle-modal-3.0.0-{%%s%%}.html"){
+    if(document.sezzleDefaultModalVersion === "sezzle-modal-3.0.0-{%%s%%}.html" || document.sezzleDefaultModalVersion === "sezzle-modal-3.0.1-{%%s%%}.html"){
```
Comment (hardikk08) [inline src/classBased/sezzle.js:37]: what is this?

**Review by hardikk08 — APPROVED** [review]

File: src/classBased/sezzle.js (hunk 1 above)
Comment (aholdahl) [reply src/classBased/sezzle.js:37]: For language refactor, since translation object is only available for these two versions.

**Pull request merged**